masonry/src/widgets/flex.rs

```diff
@@ -1189,7 +1188,6 @@ impl Widget for Flex {
 
                     let child_pos: Point = self.direction.pack(major, child_minor_offset).into();
                     ctx.place_child(widget, child_pos);
-                    child_paint_rect = child_paint_rect.union(ctx.widget_state.paint_rect());
```

Comment:
Was this (unused) `child_paint_rect` being accumulated for anything or is vestigial?

Reply:
It was used in the call to set_paint_insets, which itself was vestigial.